app/assets/javascripts/select-kit/addon/components/toolbar-popup-menu-options.js

```diff
@@ -11,15 +12,27 @@ export default DropdownSelectBoxComponent.extend({
     autoFilterable: false,
     preventHeaderFocus: true,
     customStyle: true,
+    titleProperty: "title",
+    labelProperty: "label",
   },
 
   modifyContent(contents) {
     return contents
       .map((content) => {
         if (content.condition) {
+          let label;
+          if (content.label) {
+            label = I18n.t(content.label);
+            if (content.shortcut) {
+              label += ` <kbd class="shortcut">${PLATFORM_KEY_MODIFIER}+${content.shortcut}</kbd>`;
+            }
+          }
+
           return {
             icon: content.icon,
-            name: I18n.t(content.label),
+            label,
+            title: content.title ? I18n.t(content.title) : null,
+            name: content.name,
```

Comment:
In this case it's better to use label.

SelectKit does all kind of assumptions on name/label/title... because we had a lot of history on naming in a lot of different ways.

Everything work with just name, but ideally label + title is better as it gives you more control. It allows me to show the kbd in the label, but when hover with the mouse it just  shows a different text with no shortcut (and no html).